pages/contact.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>contact</title>
+    <link rel="stylesheet" href="../build/tailwind.css" />
+</head>
+<body>
+    <div class="navbar">
+        <img class="logo" src="/img/logo.png" alt="logo"/>
+        <div class="navbar__links">
+          <a class="navbar__link" href="#"> Home </a>
+          <a class="navbar__link" href="/pages/product.html"> Product </a>
+          <a class="navbar__link" href="/pages/price.html">Price</a>
+          <a class="navbar__link" href="">About us</a>
+          <a class="navbar__link" href="#">Blog</a>
+          <a class="navbar__link" href="#">Contact</a>
+          <button class="btn">Free trial</button>
+        </div>
+      </div>
+      <div class="contact">
+
+   
+      <h1 class="title">Get in touch with us</h1>
+      <div class="contact-content">
+<div class="contact-content__input">
+<h1 class="title title--small">Drop us a message</h1>
+<p class="description">We will get back to you as soon as possible.</p>
+<form class="contact-form">
+<input class="contact__input" placeholder="Full Name">
+<input class="contact__input" placeholder="Full Name">
+<input class="contact__input contact__input--medium" placeholder="Full Name">
+<input class="contact__input contact__input--medium" placeholder="Full Name">
+<input class="contact__input contact__input--large" placeholder="Full Name">
+<button class="btn btn--large">Send</button>
+</form>
+</div>
+<div class="contact-content__info">
+    <div class="why-us__content">
```

Comment:
Also here there is an issue of using a block of `why-us` which does not exist yet! Create it or change it. Also, remember to change all the children.